examples/blog/src/layouts/BlogPost.astro

```diff
@@ -18,7 +18,8 @@ const {
 } = Astro.props;
 ---
 
-<html>
+<!DOCTYPE html>
```

Comment:
Astro adds `<!DOCTYPE html>` to pages for you, so I think I’d err on the side of leaving this out. (Notwithstanding apparently some templates already having it 😅 — maybe we need another PR removing them so they’re consistent, but certainly wouldn’t add one.)

```suggestion
```